deploy/backplane/mkrts/redhat-kas-fleetshard/20-mkrts-fleetshard-admins.SubjectPermission.yml

```diff
@@ -1,12 +1,14 @@
 apiVersion: managed.openshift.io/v1alpha1
 kind: SubjectPermission
 metadata:
-  name: backplane-mkrts-admins
+  name: backplane-mkrts-users
   namespace: openshift-rbac-permissions
 spec:
+  clusterPermissions:
+  - backplane-mkrts-readers-cluster
```

Comment:
Should this be backplane-mkrts-users-project? ie, the cut-down permissions we want to apply to the base user?

We are setting that below under permissions so I'm unsure on this clusterPermissions block.